### ISSUE TITLE
Module arcade.examples.gui_text_button crash, coin_01.png not found

### ISSUE THREAD

**Issue by owlnical** [pvcraven/arcade#284]
## Bug Report

Starting the [Text Button example](http://arcade.academy/examples/gui_text_button.html#gui-text-button) with `python3 -m arcade.examples.gui_text_button` crashes the module. Same result when I try to start it with `python3 <absolute path>`. Changing to the directory where it's located and executing `python3  gui_text_button.py` works as expected.

### Actual behavior:

```
$ python3 -m arcade.examples.gui_text_button
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/examples/gui_text_button.py", line 228, in <module>
    main()
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/examples/gui_text_button.py", line 223, in main
    game.setup()
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/examples/gui_text_button.py", line 155, in setup
    coin = arcade.Sprite("images/coin_01.png", 0.25)
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/sprite.py", line 142, in __init__
    image_width, image_height)
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/draw_commands.py", line 286, in load_texture
    source_image = PIL.Image.open(file_name)
  File "/home/USERNAME/.local/lib/python3.6/site-packages/PIL/Image.py", line 2609, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'images/coin_01.png'
```

```
$ python3 ~/.local/lib/python3.6/site-packages/arcade/examples/gui_text_button.py
Traceback (most recent call last):
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/examples/gui_text_button.py", line 228, in <module>
    main()
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/examples/gui_text_button.py", line 223, in main
    game.setup()
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/examples/gui_text_button.py", line 155, in setup
    coin = arcade.Sprite("images/coin_01.png", 0.25)
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/sprite.py", line 142, in __init__
    image_width, image_height)
  File "/home/USERNAME/.local/lib/python3.6/site-packages/arcade/draw_commands.py", line 286, in load_texture
    source_image = PIL.Image.open(file_name)
  File "/home/USERNAME/.local/lib/python3.6/site-packages/PIL/Image.py", line 2609, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'images/coin_01.png'
```

This works:

```
$ cd  ~/.local/lib/python3.6/site-packages/arcade/examples
$ python3 gui_text_button.py 
```

### Expected behavior:

The example should be started.

### Steps to reproduce/example code:

    python3 -m arcade.examples.gui_text_button

### Version

```
$ pip3 show arcade
Name: arcade
Version: 1.3.7
Summary: Arcade Game Development Library
Home-page: http://arcade.academy
Author: Paul Vincent Craven
Author-email: paul.craven@simpson.edu
License: MIT
Location: /home/USERNAME/.local/lib/python3.6/site-packages
Requires: pyglet, pillow
```

```
$ apt show python3
Package: python3
Version: 3.6.7-1
Priority: optional
Section: python
Source: python3-defaults
Maintainer: Matthias Klose <doko@debian.org>
Installed-Size: 191 kB
Provides: python3-profiler
Pre-Depends: python3-minimal (= 3.6.7-1)
Depends: python3.6 (>= 3.6.7-1~), libpython3-stdlib (= 3.6.7-1)
Suggests: python3-doc (>= 3.6.7-1), python3-tk (>= 3.6.7-1~), python3-venv (>= 3.6.7-1)
Replaces: python3-minimal (<< 3.1.2-2)
Homepage: https://www.python.org/
Tag: devel::interpreter, devel::lang:python, devel::library,
 implemented-in::c, implemented-in::python, role::devel-lib,
 role::program, role::shared-lib
Cnf-Extra-Commands: python
Cnf-Priority-Bonus: 5
Download-Size: 61.4 kB
APT-Manual-Installed: no
APT-Sources: http://ftp.se.debian.org/debian testing/main amd64 Packages
```

**Comment by owlnical** [comment]
```
$ python3 -m pyglet.info
Python
------------------------------------------------------------------------------
sys.version: 3.6.8rc1 (default, Dec 12 2018, 06:12:40) 
[GCC 8.2.0]
sys.platform: linux
sys.maxint: 9223372036854775807
os.getcwd(): /home/USERNAME/.local/lib/python3.6/site-packages/arcade/examples

pyglet
------------------------------------------------------------------------------
pyglet.version: 1.3.2
pyglet.compat_platform: linux
pyglet.__file__: /home/USERNAME/.local/lib/python3.6/site-packages/pyglet/__init__.py
pyglet.options['audio'] = ('directsound', 'pulse', 'openal', 'silent')
pyglet.options['font'] = ('gdiplus', 'win32')
pyglet.options['debug_font'] = False
pyglet.options['debug_gl'] = True
pyglet.options['debug_gl_trace'] = False
pyglet.options['debug_gl_trace_args'] = False
pyglet.options['debug_graphics_batch'] = False
pyglet.options['debug_lib'] = False
pyglet.options['debug_media'] = False
pyglet.options['debug_texture'] = False
pyglet.options['debug_trace'] = False
pyglet.options['debug_trace_args'] = False
pyglet.options['debug_trace_depth'] = 1
pyglet.options['debug_trace_flush'] = True
pyglet.options['debug_win32'] = False
pyglet.options['debug_x11'] = False
pyglet.options['graphics_vbo'] = True
pyglet.options['shadow_window'] = True
pyglet.options['vsync'] = None
pyglet.options['xsync'] = True
pyglet.options['xlib_fullscreen_override_redirect'] = False
pyglet.options['darwin_cocoa'] = False
pyglet.options['search_local_libs'] = True

pyglet.window
------------------------------------------------------------------------------
platform: <pyglet.window.Platform object at 0x7f78c68d1860>
display: <pyglet.canvas.xlib.XlibDisplay object at 0x7f78c660f630>
screens[0]: XlibScreen(display=<pyglet.canvas.xlib.XlibDisplay object at 0x7f78c660f630>, x=0, y=0, width=1920, height=1080, xinerama=0)
config['double_buffer'] = 1
config['stereo'] = 0
config['buffer_size'] = 24
config['aux_buffers'] = 0
config['sample_buffers'] = 0
config['samples'] = 0
config['red_size'] = 8
config['green_size'] = 8
config['blue_size'] = 8
config['alpha_size'] = 0
config['depth_size'] = 32
config['stencil_size'] = 0
config['accum_red_size'] = 0
config['accum_green_size'] = 0
config['accum_blue_size'] = 0
config['accum_alpha_size'] = 0
config['major_version'] = None
config['minor_version'] = None
config['forward_compatible'] = None
config['debug'] = None
context: XlibContextARB()

window.context._info
------------------------------------------------------------------------------
gl_info.get_version(): 3.1 Mesa 18.2.6
gl_info.get_vendor(): VMware, Inc.
gl_info.get_renderer(): llvmpipe (LLVM 7.0, 256 bits)
gl_info.get_extensions():
   GL_AMD_conservative_depth
   GL_AMD_draw_buffers_blend
   GL_AMD_seamless_cubemap_per_texture
   GL_AMD_shader_stencil_export
   GL_AMD_shader_trinary_minmax
   GL_ANGLE_texture_compression_dxt3
   GL_ANGLE_texture_compression_dxt5
   GL_APPLE_packed_pixels
   GL_ARB_ES2_compatibility
   GL_ARB_ES3_compatibility
   GL_ARB_arrays_of_arrays
   GL_ARB_base_instance
   GL_ARB_blend_func_extended
   GL_ARB_buffer_storage
   GL_ARB_clear_buffer_object
   GL_ARB_clear_texture
   GL_ARB_clip_control
   GL_ARB_color_buffer_float
   GL_ARB_compatibility
   GL_ARB_compressed_texture_pixel_storage
   GL_ARB_conditional_render_inverted
   GL_ARB_conservative_depth
   GL_ARB_copy_buffer
   GL_ARB_copy_image
   GL_ARB_cull_distance
   GL_ARB_debug_output
   GL_ARB_depth_buffer_float
   GL_ARB_depth_clamp
   GL_ARB_depth_texture
   GL_ARB_draw_buffers
   GL_ARB_draw_buffers_blend
   GL_ARB_draw_elements_base_vertex
   GL_ARB_draw_indirect
   GL_ARB_draw_instanced
   GL_ARB_enhanced_layouts
   GL_ARB_explicit_attrib_location
   GL_ARB_explicit_uniform_location
   GL_ARB_fragment_coord_conventions
   GL_ARB_fragment_layer_viewport
   GL_ARB_fragment_program
   GL_ARB_fragment_program_shadow
   GL_ARB_fragment_shader
   GL_ARB_framebuffer_object
   GL_ARB_framebuffer_sRGB
   GL_ARB_get_program_binary
   GL_ARB_get_texture_sub_image
   GL_ARB_half_float_pixel
   GL_ARB_half_float_vertex
   GL_ARB_instanced_arrays
   GL_ARB_internalformat_query
   GL_ARB_internalformat_query2
   GL_ARB_invalidate_subdata
   GL_ARB_map_buffer_alignment
   GL_ARB_map_buffer_range
   GL_ARB_multi_bind
   GL_ARB_multi_draw_indirect
   GL_ARB_multisample
   GL_ARB_multitexture
   GL_ARB_occlusion_query
   GL_ARB_occlusion_query2
   GL_ARB_pipeline_statistics_query
   GL_ARB_pixel_buffer_object
   GL_ARB_point_parameters
   GL_ARB_point_sprite
   GL_ARB_polygon_offset_clamp
   GL_ARB_program_interface_query
   GL_ARB_provoking_vertex
   GL_ARB_robustness
   GL_ARB_sampler_objects
   GL_ARB_seamless_cube_map
   GL_ARB_seamless_cubemap_per_texture
   GL_ARB_separate_shader_objects
   GL_ARB_shader_bit_encoding
   GL_ARB_shader_objects
   GL_ARB_shader_stencil_export
   GL_ARB_shader_subroutine
   GL_ARB_shader_texture_lod
   GL_ARB_shading_language_100
   GL_ARB_shading_language_420pack
   GL_ARB_shading_language_packing
   GL_ARB_shadow
   GL_ARB_stencil_texturing
   GL_ARB_sync
   GL_ARB_texture_border_clamp
   GL_ARB_texture_buffer_object
   GL_ARB_texture_buffer_object_rgb32
   GL_ARB_texture_buffer_range
   GL_ARB_texture_compression
   GL_ARB_texture_compression_bptc
   GL_ARB_texture_compression_rgtc
   GL_ARB_texture_cube_map
   GL_ARB_texture_cube_map_array
   GL_ARB_texture_env_add
   GL_ARB_texture_env_combine
   GL_ARB_texture_env_crossbar
   GL_ARB_texture_env_dot3
   GL_ARB_texture_float
   GL_ARB_texture_gather
   GL_ARB_texture_mirror_clamp_to_edge
   GL_ARB_texture_mirrored_repeat
   GL_ARB_texture_multisample
   GL_ARB_texture_non_power_of_two
   GL_ARB_texture_query_levels
   GL_ARB_texture_query_lod
   GL_ARB_texture_rectangle
   GL_ARB_texture_rg
   GL_ARB_texture_rgb10_a2ui
   GL_ARB_texture_stencil8
   GL_ARB_texture_storage
   GL_ARB_texture_storage_multisample
   GL_ARB_texture_swizzle
   GL_ARB_texture_view
   GL_ARB_timer_query
   GL_ARB_transform_feedback2
   GL_ARB_transform_feedback3
   GL_ARB_transform_feedback_instanced
   GL_ARB_transform_feedback_overflow_query
   GL_ARB_transpose_matrix
   GL_ARB_uniform_buffer_object
   GL_ARB_vertex_array_bgra
   GL_ARB_vertex_array_object
   GL_ARB_vertex_attrib_binding
   GL_ARB_vertex_buffer_object
   GL_ARB_vertex_program
   GL_ARB_vertex_shader
   GL_ARB_vertex_type_10f_11f_11f_rev
   GL_ARB_vertex_type_2_10_10_10_rev
   GL_ARB_viewport_array
   GL_ARB_window_pos
   GL_ATI_blend_equation_separate
   GL_ATI_draw_buffers
   GL_ATI_fragment_shader
   GL_ATI_separate_stencil
   GL_ATI_texture_compression_3dc
   GL_ATI_texture_env_combine3
   GL_ATI_texture_float
   GL_ATI_texture_mirror_once
   GL_EXT_abgr
   GL_EXT_bgra
   GL_EXT_blend_color
   GL_EXT_blend_equation_separate
   GL_EXT_blend_func_separate
   GL_EXT_blend_minmax
   GL_EXT_blend_subtract
   GL_EXT_compiled_vertex_array
   GL_EXT_copy_texture
   GL_EXT_draw_buffers2
   GL_EXT_draw_instanced
   GL_EXT_draw_range_elements
   GL_EXT_fog_coord
   GL_EXT_framebuffer_blit
   GL_EXT_framebuffer_multisample
   GL_EXT_framebuffer_multisample_blit_scaled
   GL_EXT_framebuffer_object
   GL_EXT_framebuffer_sRGB
   GL_EXT_gpu_program_parameters
   GL_EXT_multi_draw_arrays
   GL_EXT_packed_depth_stencil
   GL_EXT_packed_float
   GL_EXT_packed_pixels
   GL_EXT_pixel_buffer_object
   GL_EXT_point_parameters
   GL_EXT_polygon_offset_clamp
   GL_EXT_provoking_vertex
   GL_EXT_rescale_normal
   GL_EXT_secondary_color
   GL_EXT_separate_specular_color
   GL_EXT_shader_integer_mix
   GL_EXT_shadow_funcs
   GL_EXT_stencil_two_side
   GL_EXT_stencil_wrap
   GL_EXT_subtexture
   GL_EXT_texture
   GL_EXT_texture3D
   GL_EXT_texture_array
   GL_EXT_texture_compression_dxt1
   GL_EXT_texture_compression_latc
   GL_EXT_texture_compression_rgtc
   GL_EXT_texture_compression_s3tc
   GL_EXT_texture_cube_map
   GL_EXT_texture_edge_clamp
   GL_EXT_texture_env_add
   GL_EXT_texture_env_combine
   GL_EXT_texture_env_dot3
   GL_EXT_texture_integer
   GL_EXT_texture_lod_bias
   GL_EXT_texture_mirror_clamp
   GL_EXT_texture_object
   GL_EXT_texture_rectangle
   GL_EXT_texture_sRGB
   GL_EXT_texture_sRGB_decode
   GL_EXT_texture_shared_exponent
   GL_EXT_texture_snorm
   GL_EXT_texture_swizzle
   GL_EXT_timer_query
   GL_EXT_transform_feedback
   GL_EXT_vertex_array
   GL_EXT_vertex_array_bgra
   GL_IBM_multimode_draw_arrays
   GL_IBM_rasterpos_clip
   GL_IBM_texture_mirrored_repeat
   GL_INGR_blend_func_separate
   GL_KHR_context_flush_control
   GL_KHR_debug
   GL_KHR_no_error
   GL_KHR_texture_compression_astc_ldr
   GL_MESA_pack_invert
   GL_MESA_shader_integer_functions
   GL_MESA_texture_signed_rgba
   GL_MESA_window_pos
   GL_MESA_ycbcr_texture
   GL_NV_blend_square
   GL_NV_conditional_render
   GL_NV_depth_clamp
   GL_NV_fog_distance
   GL_NV_light_max_exponent
   GL_NV_packed_depth_stencil
   GL_NV_primitive_restart
   GL_NV_texgen_reflection
   GL_NV_texture_env_combine4
   GL_NV_texture_rectangle
   GL_OES_EGL_image
   GL_OES_read_format
   GL_S3_s3tc
   GL_SGIS_generate_mipmap
   GL_SGIS_texture_border_clamp
   GL_SGIS_texture_edge_clamp
   GL_SGIS_texture_lod
   GL_SUN_multi_draw_arrays

pyglet.gl.glu_info
------------------------------------------------------------------------------
glu_info.get_version(): 1.3
glu_info.get_extensions():
   GLU_EXT_nurbs_tessellator
   GLU_EXT_object_space_tess

pyglet.gl.glx_info
------------------------------------------------------------------------------
context.is_direct(): 1
glx_info.get_server_vendor(): SGI
glx_info.get_server_version(): 1.4
glx_info.get_server_extensions():
   GLX_ARB_context_flush_control
   GLX_ARB_create_context
   GLX_ARB_create_context_no_error
   GLX_ARB_create_context_profile
   GLX_ARB_fbconfig_float
   GLX_ARB_framebuffer_sRGB
   GLX_ARB_multisample
   GLX_EXT_create_context_es_profile
   GLX_EXT_create_context_es2_profile
   GLX_EXT_fbconfig_packed_float
   GLX_EXT_framebuffer_sRGB
   GLX_EXT_import_context
   GLX_EXT_libglvnd
   GLX_EXT_no_config_context
   GLX_EXT_texture_from_pixmap
   GLX_EXT_visual_info
   GLX_EXT_visual_rating
   GLX_MESA_copy_sub_buffer
   GLX_OML_swap_method
   GLX_SGI_make_current_read
   GLX_SGIS_multisample
   GLX_SGIX_fbconfig
   GLX_SGIX_pbuffer
   GLX_SGIX_visual_select_group
glx_info.get_client_vendor(): Mesa Project and SGI
glx_info.get_client_version(): 1.4
glx_info.get_client_extensions():
   GLX_ARB_context_flush_control
   GLX_ARB_create_context
   GLX_ARB_create_context_profile
   GLX_ARB_create_context_robustness
   GLX_ARB_fbconfig_float
   GLX_ARB_framebuffer_sRGB
   GLX_ARB_get_proc_address
   GLX_ARB_multisample
   GLX_EXT_buffer_age
   GLX_EXT_create_context_es2_profile
   GLX_EXT_create_context_es_profile
   GLX_EXT_fbconfig_packed_float
   GLX_EXT_framebuffer_sRGB
   GLX_EXT_import_context
   GLX_EXT_texture_from_pixmap
   GLX_EXT_visual_info
   GLX_EXT_visual_rating
   GLX_INTEL_swap_event
   GLX_MESA_copy_sub_buffer
   GLX_MESA_multithread_makecurrent
   GLX_MESA_query_renderer
   GLX_MESA_swap_control
   GLX_OML_swap_method
   GLX_OML_sync_control
   GLX_SGIS_multisample
   GLX_SGIX_fbconfig
   GLX_SGIX_pbuffer
   GLX_SGIX_visual_select_group
   GLX_SGI_make_current_read
   GLX_SGI_swap_control
   GLX_SGI_video_sync
glx_info.get_extensions():
   GLX_ARB_context_flush_control
   GLX_ARB_create_context
   GLX_ARB_create_context_profile
   GLX_ARB_fbconfig_float
   GLX_ARB_framebuffer_sRGB
   GLX_ARB_get_proc_address
   GLX_ARB_multisample
   GLX_EXT_create_context_es2_profile
   GLX_EXT_create_context_es_profile
   GLX_EXT_fbconfig_packed_float
   GLX_EXT_framebuffer_sRGB
   GLX_EXT_import_context
   GLX_EXT_texture_from_pixmap
   GLX_EXT_visual_info
   GLX_EXT_visual_rating
   GLX_MESA_copy_sub_buffer
   GLX_MESA_query_renderer
   GLX_OML_swap_method
   GLX_SGIS_multisample
   GLX_SGIX_fbconfig
   GLX_SGIX_pbuffer
   GLX_SGIX_visual_select_group
   GLX_SGI_make_current_read

pyglet.media
------------------------------------------------------------------------------
audio driver: <pyglet.media.drivers.pulse.adaptation.PulseAudioDriver object at 0x7f78c4754828>

pyglet.media.avbin
------------------------------------------------------------------------------
AVbin not available.

pyglet.media.drivers.openal
------------------------------------------------------------------------------
Library: <CDLL 'libopenal.so.1', handle 2d1d750 at 0x7f78b63376d8>
Version: 1.1
Extensions:
   ALC_ENUMERATE_ALL_EXT
   ALC_ENUMERATION_EXT
   ALC_EXT_CAPTURE
   ALC_EXT_DEDICATED
   ALC_EXT_disconnect
   ALC_EXT_EFX
   ALC_EXT_thread_local_context
   ALC_SOFT_device_clock
   ALC_SOFT_HRTF
   ALC_SOFT_loopback
   ALC_SOFT_output_limiter
   ALC_SOFT_pause_device

pyglet.input.wintab
------------------------------------------------------------------------------
WinTab not available.
Traceback (most recent call last):
  File "_ctypes/callbacks.c", line 234, in 'calling callback function'
  File "/home/USERNAME/.local/lib/python3.6/site-packages/pyglet/media/drivers/pulse/interface.py", line 365, in _state_callback
  File "/home/USERNAME/.local/lib/python3.6/site-packages/pyglet/media/drivers/pulse/interface.py", line 185, in signal
AttributeError: 'NoneType' object has no attribute 'signal'
```

**Comment by pvcraven** [comment]
Thanks. Fix pushed to master branch.

**Comment by owlnical** [comment]
Thank you for the quick fix!

**Comment by pvcraven** [comment]
Closed with 2.0 release.
